policy/modules/kernel/kernel.te

```diff
@@ -374,13 +374,20 @@ ifdef(`init_systemd',`
 	optional_policy(`
 		selinux_compute_create_context(kernel_t)
 	')
+')
 
-	optional_policy(`
-		storage_dev_filetrans_fixed_disk(kernel_t)
-		storage_setattr_fixed_disk_dev(kernel_t)
-		storage_create_fixed_disk_dev(kernel_t)
-		storage_delete_fixed_disk_dev(kernel_t)
-	')
+optional_policy(`
+	storage_dev_filetrans_fixed_disk(kernel_t)
```

Comment:
What is the non-systemd usage scenario?

Reply:
The usage scenario is other init daemons in general, see for example SysVInit:

http://www.nongnu.org/sysvinit
https://git.savannah.nongnu.org/cgit/sysvinit.git

Reply:
I understand there are other init systems, I'm asking can you provide an example where this is happening? code, script, etc.

Reply:
Usually scripts that run at bootup need to carry out operations on storage devices.

Reply:
The new permissions introduced do not seem partilarly risky when executed from a kernel context: at the end, it seems quite normal to me that the kernel should at least be able able to [gs]etattr storage devices.